src/data/faq.json

```diff
@@ -101,6 +101,166 @@
+                        ]
+                    },
+                    {
+                        "title": "Does vaccination status affect the calculation of the risk? For example, if a vaccinated person has become infected and then warns other users?",
+                        "anchor": "vaccination_certificate_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "No, do not take into account in the risk calculation whether you are vaccinated against or recovered from a corona infection. The functions are independent of each other. The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary. However, your personal vaccination status may affect your interpretation of the risk warning."
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_red_warning",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "Can I save my vaccination certificate on multiple smartphones, e.g., on my private and business phone? How can I transfer the vaccination certificate when changing device? What do I do if I lose my phone? Will the vaccination certificate be included in my backup? Can I manage the certificate in the Corona-Warn-App and the CovPass-App at the same time, or do I have to choose one of the two apps?",
+                        "anchor": "vaccination_certificate_multiple_scan_possible",
+                        "active": true,
+                        "textblock": [
+                            "There is no limit to how often you can scan the 2D barcode (QR code) of your vaccination certificates. You can add and manage the vaccination certificates on multiple devices, on a new device, on a device you have reset, and even in several apps. Therefore, a backup is not required."
+                        ]
+                    },
+                    {
+                        "title": "Can I save the vaccination certificates of relatives, such as children, in the Corona-Warn-App?",
+                        "anchor": "vaccination_certificate_relatives",
+                        "active": true,
+                        "textblock": [
+                            "Can I save the vaccination certificates of relatives, such as children, in the Corona-Warn-App?<br>With the Corona-Warn-App, you can only scan and add vaccination certificates from one person. It is not possible to add your own and additionally, for example, that of your child or partner.<br>With the CovPass-App, the digital vaccination certificates of children or partners can be stored together on a smartphone."

```

Comment:
Why is title repeated in the textblock? Can it be removed from the textblock?
"Can I save the vaccination certificates of relatives, such as children, in the Corona-Warn-App?"

Reply:
fixed